### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BDMorrell**
Changed the readme's shell code to be highlighted as code, for the next person who's trying to skim though this for downloading.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,15 @@ Ivy is an interpreter for an APL-like language. It is a plaything and a work in
 progress.
 
 Ivy has a custom domain. Do not install using github directly. Instead, run:
+	```
```
Comment (theclapp) [inline README.md]: You might as well go all the way and say ` ```sh ` to add syntax highlighting.  It's only one line, but why not.

**Comment by robpike** [comment]
Thanks for the suggestion. (No need for syntax highlighting, please.)

You'll need to sign thee copyright license agreement (see https://golang.org/doc/contribute.html#cla) before I can accept your pull request. Sorry about that.

**Comment by shurcooL** [comment]
I wanted to make some suggestions how I think the formatting can be improved even further, but it's quite hard to do via code review here. I just sent an alternative PR #43, please consider it as my code review suggestion. I've signed the copyright license agreement.

**Comment by robpike** [comment]
Superseded.

**Pull request closed without merging**